packages/medusa/src/api/routes/admin/inventory-items/list-inventory-items.ts

```diff
@@ -131,10 +132,21 @@ export default async (req: Request, res: Response) => {
 
   const inventoryItemsWithVariantsAndLocationLevels = inventoryItems.map(
     (inventoryItem) => {
+      const levels = levelsByItemId[inventoryItem.id] ?? []
```

Comment:
suggestion(nit): Would it make sense to move that to a separate util, I can see the users wanted to do the same

Reply:
yeah! I've put it into the join-levels util since it's heavily dependent on that 😄